src/docs/software/using/alphafold.md

```diff
@@ -0,0 +1,296 @@
+    user must request their own copy of the model parameters. Researchers
+    can complete the form the request form found [here][url_form].
+    
+    When filling out the request form, **you must use a personal Gmail
+    address for the Google account email address field.** Your Stanford
+    email address will not work and your request will be rejected.
+    
+2. Download the model parameters to Sherlock.
+
+    Once your access is approved by Google DeepMind, you will receive a link
+    to download the model parameters file, `af3.bin.zst`. It is recommended to
+    download the model parameters to `$HOME`. The file is ~1 GB in size.
+    
+    ```
+    $ mkdir $HOME/af3_model
+    $ cd $HOME/af3_model
+    $ wget <personal_download_link>
+    ```
+    You can make a copy of your model parameters to `$SCRATCH`, which will
+    have faster performance when running AlphaFold 3.
+    
+    ```
+    $ cp -R $HOME/af3_model $SCRATCH
+    ```
+    
+3. Clone the AlphaFold 3 GitHub Repo into `$GROUP_HOME`
+
+    ```
+    $ mkdir -p $GROUP_HOME/$USER
+    $ cd $GROUP_HOME/$USER
+    $ git clone https://github.com/google-deepmind/alphafold3.git
+    ```
+    
+4. Download the Databases to `$SCRATCH` or `$GROUP_SCRATCH`.
+
+    For faster performance, you will need a copy of the AlphaFold 3 databases
+    in `$SCRATCH` or `$GROUP_SCRATCH`. You can use the following template to
+    create a batch job for downloading the databases.
+    
+    ```
+    #!/bin/bash
+    #SBATCH -t 5:00:00
+    
+    bash fetch_databases.sh $SCRATCH/af3_db
+    ```
+    
+    This script assumes that you are submitting the job from within the 
+    GitHub repository that you cloned into `$GROUP_HOME/$USER`. Feel free to
+    modify the script if you want to use your own partition, download to a
+    different location, or submit the job from a different directory. The
+    database will take approximately 1 hour to download, you may proceed to
+    the next step while it completes.
+    
+    Although unmodified files are purged from `$SCRATCH` and `$GROUP_SCRATCH`
+    every 90 days, Stanford Research Computing maintains a copy of the
+    AlphaFold 3 databases within the Oak Common Datasets Repository. As you run
+    AlphaFold, you can include a `dcp` in your scripts that compares your
+    `$SCRATCH` databases with those on `$OAK` and automatically copies over
+    any missing files.
+    
+6. Build the AlphaFold 3 Apptainer Image
+
+    Container software like Apptainer allow researchers to easily "contain"
+    or package software and port it between compute platforms. The following
+    commands will allow you to build your own copy of AlphaFold 3 into
+    a `.sif` file, or container image.
+    
+    ```
+    $ sh_dev -m 12GB
+    
+    $ cd $GROUP_HOME/$USER/alphafold3
+    
+    $ wget https://gist.githubusercontent.com/cagancayco/a71ae18ab5044e14526e0c83becaf647/raw/9f6104366d59fa9a96bb92693cfa60930184c981/af3.def
```

Comment:
What about hosting this here in the documentation? Either by having the full `.def` file inline in the page, like we do here https://www.sherlock.stanford.edu/docs/user-guide/running-jobs/#recurring-job-example?

Ultimately, it may be even better to pre-build the container and make it available through a module or something, so users wouldn't have to build it themselves? Any downsides to that approach?